addons/website_profile/controllers/main.py

```diff
@@ -183,3 +183,25 @@ def badges(self, **kwargs):
             'badges': badges,
         })
         return request.render("website_profile.badge_main", values)
+
+    # All Users Page
+    # ---------------------------------------------------
+    def _prepare_all_users_values(self, user, position):
+        return {
+            'position': position,
+            'id': user.id,
+            'name': user.name,
+            'rank': user.rank_id.name,
+            'karma': user.karma,
+            'badge_count': len(user.badge_ids),
+        }
+
+    @http.route(['/profile/users'], type='http', auth="public", website=True)
+    def view_all_users_page(self, **post):
+        dom = [('karma', '>', 1), ('website_published', '=', True)]
+        users = request.env['res.users'].sudo().search(dom, order='karma DESC')
```

Comment:
On odoo.com, we got 1072753 of active res.users entries for now.
I'm sorry, but you gonna need to create a pager to only fetch 10 user per 10 users. Otherwise we gonna have perfomance problem.